Fixed NREs & updated WindowTests

Kira-PH opened the pull request:
Check out WindowTests!

It doesn't work - there's a missed collision. Right now my two guesses are a bad collision mask on one of the objects, or that one of them is a ChainShape (I haven't done the Chain collisions yet)

Commit:
Fixed DebugDraw
Fixed NREs
Added a "Test World" and updated WindowTests

Commit:
Merge pull request #1 from benzuk/master

Update my branch

Commit:
Fixed DebugDraw
Fixed NREs
Added a "Test World" and updated WindowTests

Commit:
Commit deletion of Controllers dir

Commit:
Merge branch 'master' of github.com:HughPH/box2d-netstandard

 Conflicts:
	tests/unit/Box2D.NetStandard.UnitTests/MathTests.cs

Commit:
Updated solution file

HughPH commented:
Actually neither is a chain shape and the collision masks look fine, so it's got to be a bug in poly/poly collisions. But it's nearly midnight here so I'll have a look tomorrow.

Commit:
Fixed additional Null Reference Exceptions
Tidied verbose calls
Set CameraView to be focused on The Action
Modified mouse wheel action for a more friendly zoom
Renamed Filter to filter to be more in-line with C++ version
Tweaked object properties in RubeGoldberg
